index.html

```diff
@@ -50,8 +50,8 @@ <h1>RDF*</h1>
         </div> 	
 
         <p>
-          <a href="https://w3c.github.io/rdf-star/">RDF* Home</a>, part of the 
-          <a href="https://www.w3.org/community/rdf-dev/">RDF dev community group.</a>
+          <a href="https://w3c.github.io/rdf-star/">Home of RDF-Star (née RDF*)</a>, a part of the 
```

Comment:
As per my previous comment...
```suggestion
          <a href="https://w3c.github.io/rdf-star/">Home of RDF-* (pronounced "RDF star")</a>, a part of the 
```

Reply:
sorry, copy-paste error there. meant to take your suggestion.